pkg/configtx/constants.go

```diff
@@ -111,9 +85,5 @@ const (
 	// SignaturePolicyType is the 'Type' string for signature policies.
 	SignaturePolicyType = "Signature"
 
-	// ConsensusStateNormal indicates normal orderer operation.
-	ConsensusStateNormal ConsensusState = "STATE_NORMAL"
-
-	// ConsensusStateMaintenance indicates the orderer is in consensus type migration.
-	ConsensusStateMaintenance ConsensusState = "STATE_MAINTENANCE"
+	ordererAdminsPolicyName = "/Channel/Orderer/Admins"
```

Comment:
consider moving this too? Although I can see it staying as well

Reply:
I did consider this, however this unexported constant is used in multiple files (other than orderer subpackage and examples).